config/tracks/ks_nordschleife.ini

```diff
@@ -540,7 +538,7 @@ CONDITION=SEASON_SUMMER
 ACTIVE=1
 VISIBILITY_LEVEL=0
 DESCRIPTION=summer settings for road + curbs + walls + stuff
-MATERIALS=concrete, park-dirt, tavr, armco, pitlane-multimap?, Vehicles, box, asphalt-, pitlane-multilayer, parkAO-shad, t13, karussell, Kleinekarussell, concgrid, brickfloor?, wall1, curb-new, tyres, asph shad, asph shad dark, pavement-brick
```

Comment:
Removed box mesh from summer settings as it renders GP cylindrical windows in a milky shade during summer with CSP windows.